### ISSUE TITLE
Add 'Reject and block user' button to invites

### ISSUE THREAD

**Issue by frebib** [element-hq/element-x-android#3614]
#### What would you like to do?
Add a way to ignore an invite and block a user in one click

#### Why would you like to do it?
Because dealing with spam/abuse in Matrix is hard and this makes it just a little bit easier.
Element Web has it and it's very useful

#### How would you like to achieve it?
With my finger

Maybe a 3rd button: `(Accept)` `(Reject)` `(Block)`
Could do with some design input

### Have you considered any alternatives?

Ignoring the invite, then finding the user somewhere and blocking them separately. It's considerably harder and slower doing that though

### Are you willing to provide a PR?

No

**Comment by Andoriax** [comment]
Yes, this is a great notion that makes to much sense to even consider not jumping on it.
And I too would like to achieve it with my finger.

**Comment by rom4nik** [comment]
FYI this button was added in https://github.com/element-hq/element-x-android/pull/4353/commits/82dc29b91c0a49301265ac4c4345eceb96d5c93a.
